### PR TITLE
Fixes Fancy Sofa

### PR DESCRIPTION
**Pull request by KathrinBailey**
# About The Pull Request
• Fixes back sprite of corporate/fancy sofa having a janky pattern.
• Fixes the back (north-facing) sprite for the left/right corporate sofas being the wrong way around.

## Why It's Good For The Game
bug bad

## A Port?
https://github.com/tgstation/tgstation/pull/69594
https://github.com/tgstation/tgstation/pull/62417

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
fix: Corporate sofas are no longer janky on the rear-facing sprite.
/:cl:

### CODE REVIEW

File: modular_splurt/code/game/objects/structures/bed_chairs/sofa.dm [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/obj/structure/chair/sofa/corp
+	icon = 'modular_splurt/icons/obj/sofa.dmi'
+
+/obj/structure/chair/sofa/corp/corner/handle_layer() //only the armrest/back of this chair should cover the mob.
+	return
```
Comment (BongaTheProto) [inline modular_splurt/code/game/objects/structures/bed_chairs/sofa.dm:5]: Small issue
dreamseeker_GoQZAeW92k

**Review by BongaTheProto — CHANGES_REQUESTED** [review]

File: modular_splurt/code/game/objects/structures/bed_chairs/sofa.dm (hunk 1 above)
Comment (KathrinBailey) [reply modular_splurt/code/game/objects/structures/bed_chairs/sofa.dm]: hgnngghhstt'veee
I literally tested this but I guess my VSC must've not finished compiling or something ;_;

**Comment by BongaTheProto** [comment]
Welp now there's another but idk if it was already like this
image

**Comment by KathrinBailey** [comment]
> Welp now there's another but idk if it was already like this image

Yup this is what it was like beforehand

**Pull request merged**